test/parallel/test-cluster-dgram-1.js

```diff
@@ -28,25 +28,37 @@ const assert = require('assert');
 const cluster = require('cluster');
 const dgram = require('dgram');
 
+function getPort(cb) {
+  const s = dgram.createSocket({ type: 'udp4', reuseAddr: true });
+  s.bind(0, () => {
+    const port = s.address().port;
+    s.close();
+    cb(null, port);
```

Comment:
This port assignment was relay on the suggestion of @benjamingr  in
https://github.com/nodejs/node/pull/12426#issuecomment-294296958

Maybe the right solution is to close this port only in the end of the test.